### ISSUE TITLE
CharString and Document seem need DocDiff:: prefix on bin/docdiff

### ISSUE THREAD

**Issue by kmuto** [hisashim/docdiff#14]
I noticed docdiff failed to work due to

```
/usr/bin/docdiff:146:in `<main>': uninitialized constant CharString (NameError)
```

Does it need DocDiff:: prefix for CharString?
Line 169 and 170, "Document" also.

**Comment by hisashim** [comment]
Oops, my fix to #3 was sloppy, thank you and excuse me.
I believe 82abaee fixes this.

**Comment by hisashim** [comment]
Oops, should have closed this years ago. Thanks.
